Note that wifi SSID is case sensitive. (#1270)
advancedtelematic/aktualizr: docs/posix-secondaries-bitbaking.adoc

|`RPI_WIFI_ENABLE`
|`"0"`
|A flag to enable/disable (default) WiFi on RPi board.

|`RPI_WIFI_SSID`
|`N/A (mandatory if wifi is enabled)`

> |ID of a WiFi network to connect to. Be aware this is case sensitive!

Reviewer: I dunno if we need to shout at people...

```suggestion
|SSID of a WiFi network to connect to. The SSID is case sensitive.
```
